04_sql.Rmd

```diff
@@ -81,69 +96,99 @@ Once the installation is complete, you can click `Finish` to exit the installer.
 
 ### Installation troubleshooting
 
-If you are not able to successfully install DBeaver on your own, please attend DataLab's Virtual Office Hours. Click [here](https://datalab.ucdavis.edu/office-hours/){rel="noopener noreferrer" target="_blank"} for more information and to receive a Zoom link.
+If you are not able to successfully install DBeaver on your own, please attend 
+DataLab's Virtual Office Hours. Click 
+[here](https://datalab.ucdavis.edu/office-hours/){rel="noopener noreferrer" target="_blank"} 
+for more information and to receive a Zoom link.
 
 ## DB Browser
```

Comment:
In the future, we might want to have a section on Database Software and have DB Browser and SQLite under that.